README.md

```diff
@@ -99,7 +99,7 @@ a disk named /dev/sda, you would run the following command to partition, format
 and mount the disk.
 
 ```console
-sudo nix --experimental-features "nix-command flakes" run github:nix-community/disko/latest -- --mode disko /tmp/disk-config.nix
+sudo nix run --refresh 'github:nix-community/disko/latest#disko-install' -- --mode disko /tmp/disk-config.nix
```

Comment:
```suggestion
sudo nix run disko/latest#disko-install -- --mode disko /tmp/disk-config.nix
```

Reply:
I wouldn't rely on the registry anymore. Many people including me are overriding that with local versions now.

Reply:
I believe when you specify a ref, then it won't use the pinned registry entry locally. For context, I'm using `flake-utils-plus` to pin my registry:

```
$ nix registry list
system flake:disko path:/nix/store/4p84shmrgsdbq2j0dfa2cbqfbxxm05h7-source?lastModified=1728334376&narHash=sha256-CTKEKPzD/j8FK6H4DO3EjyixZd3HHvgAgfnCwpGFP5c%3D&rev=d39ee334984fcdae6244f5a8e6ab857479cbaefe
global flake:disko github:nix-community/disko
```

```
$ nix flake metadata disko
Resolved URL:  path:/nix/store/4p84shmrgsdbq2j0dfa2cbqfbxxm05h7-source?lastModified=1728334376&narHash=sha256-CTKEKPzD/j8FK6H4DO3EjyixZd3HHvgAgfnCwpGFP5c%3D&rev=d39ee334984fcdae6244f5a8e6ab857479cbaefe
Locked URL:    path:/nix/store/4p84shmrgsdbq2j0dfa2cbqfbxxm05h7-source?lastModified=1728334376&narHash=sha256-CTKEKPzD/j8FK6H4DO3EjyixZd3HHvgAgfnCwpGFP5c%3D&rev=d39ee334984fcdae6244f5a8e6ab857479cbaefe
Description:   Disko - declarative disk partitioning
Path:          /nix/store/4p84shmrgsdbq2j0dfa2cbqfbxxm05h7-source
Revision:      d39ee334984fcdae6244f5a8e6ab857479cbaefe
Last modified: 2024-10-08 07:52:56
Fingerprint:   67a8c3587b0de92d19455ea675f0b7e2573bef80b8f9d9b5612df0edd072164b
Inputs:
└───nixpkgs: github:NixOS/nixpkgs/b833ff01a0d694b910daca6e2ff4a3f26dee478c?narHash=sha256-tLGCFEFTB5TaOKkpfw3iYT9dnk4awTP/q4w%2BROpMfuw%3D (2024-09-01 12:44:31)
```

```
$ nix flake metadata disko/latest
Resolved URL:  github:nix-community/disko/latest
Locked URL:    github:nix-community/disko/ff0a471763faaaca1859fd6de80f44fa0fce91a6?narHash=sha256-O0QVhsj9I/hmcIqJ4qCqFyzvjYL%2BdtzJP0C5MFd8O/Y%3D
Description:   Disko - declarative disk partitioning
Path:          /nix/store/4k3ka3z47d5aysf8wijvfn9hnv7nvsal-source
Revision:      ff0a471763faaaca1859fd6de80f44fa0fce91a6
Last modified: 2024-10-12 06:02:24
Fingerprint:   b11e370bd711abe0f9ceb7c64b0d6738b4327e7ee6384d0f6d1d17250d33aa83
Inputs:
└───nixpkgs: github:NixOS/nixpkgs/b833ff01a0d694b910daca6e2ff4a3f26dee478c?narHash=sha256-tLGCFEFTB5TaOKkpfw3iYT9dnk4awTP/q4w%2BROpMfuw%3D (2024-09-01 12:44:31)
```

Reply:
Ah that's an interesting point. I don't really like relying on the registry because it has been called for to be removed. But I guess if that happens we can still change it. And using it makes the command less intimidating, which is always nice for first-time users.